plugins/scaffolder/src/components/JobStatusModal/JobStatusModal.tsx

```diff
@@ -23,18 +23,23 @@ import {
   LinearProgress,
 } from '@material-ui/core';
 
-import React, { useCallback, useState } from 'react';
+import React, { useCallback } from 'react';
 import { Job } from '../../types';
 import { JobStage } from '../JobStage/JobStage';
 
 type Props = {
   job: Job | null;
   toCatalogLink?: string;
+  open: boolean;
+  setOpen: (newState: boolean) => void;
```

Comment:
Feeling it makes more sense to make this `onClose`, you usually don't pass down setters as props